lib/delve.js

```diff
@@ -73,7 +73,7 @@ function run(file, method) {
 				dlvConnection = conn;
 				addOutputMessage("delve", `Started delve with "${file}" with "${method}"`);
 				store.dispatch({ type: "SET_STATE", state: "started" });
-
+				command("continue");
```

Comment:
Call it after adding all breakpoints is done.
Calling `continue` before adding them will (in the worst case) run the whole program / test til the end without hitting any breakpoint!

So this might look something like this (untested):

```js
Promise.all(
	getBreakpoints().map((bp) => {
		return addBreakpoint(bp.file, bp.line);
	});
).then(() => {
	command("continue");
});
```